cmd/process-agent/releasenotes/notes/Simple-settings-refactor-94f5a73cd489f5a1.yaml

```diff
@@ -0,0 +1,8 @@
+---
+prelude: Process Agent Config Changes
+features:
+  - Added `process_config.allow_rt` setting, which allows the user to toggle the realtime checks. It is true by default.
+deprecations:
+  - Removed `LOG_TO_CONSOLE` and `DD_LOGS_STDOUT` environment variables. Please use `DD_LOG_TO_CONSOLE` instead
+  - Removed `LOG_LEVEL` environment variable. Please use `DD_LOG_LEVEL` instead.
+  - Removed `warning` as a valid option for `DD_LOG_LEVEL`. Please use `warn` instead
```

Comment:
Out of curiosity, why are we removing it? Does core agent support it?

Reply:
The core agent said they didn't want to support it, and I'm pretty sure @xornivore gave me the go ahead to remove it.

Reply:
The risk is minimal to remove it, and it increases consistency.